src/Generator.Bind/Generators/Bases/GeneratorSettingsBase.cs

```diff
@@ -40,6 +40,9 @@ internal abstract class GeneratorSettingsBase : IGeneratorSettings
         /// <inheritdoc/>
         public abstract string APITypemap { get; }
 
+        /// <inheritdoc/>
+        public abstract string LicenseText { get; }
```

Comment:
The settings class should not contain actual data - if a configurable license is required, it should be a path to a file instead. 

Beyond that, why is this needed? Was the license not inserted properly?

Reply:
The license text was being read from a file every single time we wrote a file. This is a huge amount of disk IO that we've now avoided by reading it once.

Reply:
Ah, that's a good reason. However, it still doesn't belong in a settings class. It should be sourced somewhere else.